snowflake/table_snowflake_resource_monitor.go

```diff
@@ -0,0 +1,92 @@
+package snowflake
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableSnowflakeResourceMonitor(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "snowflake_resource_monitor",
+		Description: "Lists all the resource monitors in your account for which you have access privileges.",
+		List: &plugin.ListConfig{
+			Hydrate: listSnowflakeResourceMonitors,
+		},
+		Columns: snowflakeColumns([]*plugin.Column{
+			{Name: "name", Type: proto.ColumnType_STRING, Description: "Name for warehouse."},
+			{Name: "credit_quota", Type: proto.ColumnType_DOUBLE, Description: "Monthly credit quota for the resource monitor."},
```

Comment:
Is this column a money value, i.e., in USD, CAD, or something else?

Reply:
This is not a money value, see https://docs.snowflake.com/en/user-guide/resource-monitors.html#credit-quota

Reply:
Thanks @paulsewardasc , and for the credit values, can they have fractions, e.g., 300.05, or are they always whole numbers? From the doc you linked, their examples are whole numbers, but they may have just picked those arbitrarily.

Reply:
Resolving this conversation, as DOUBLE is the correct type as per Snowflake's definition mentioned in https://github.com/turbot/steampipe-plugin-snowflake/pull/8#discussion_r873694519